quickjs/testing/src/jvmMain/kotlin/app/cash/quickjs/testing/suspendingJvm.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.quickjs.testing
+
+import app.cash.quickjs.KtBridge
+
+val KtBridge.jsSuspendingEchoService: SuspendingEchoService
+  get() = get("jsSuspendingEchoService", EchoJsAdapter)
+
+class JvmSuspendingEchoService(
+  private val greeting: String
+) : SuspendingEchoService {
+  override suspend fun suspendingEcho(request: EchoRequest): EchoResponse {
+    return EchoResponse("$greeting from the suspending JVM, ${request.message}")
```

Comment:
I'm a bit worried that none of the tests actually suspend. This means that the result is delivered synchronously through the return value rather than through the Continuation parameter. Once we get `setTimeout`, all of these should add a `yield()` or `delay(1)` or something.

Reply:
100% agree.